app/job/rest.py

```diff
@@ -185,7 +186,7 @@ def create_job(service_id):
 
     elif template.template_type == EMAIL_TYPE:
         check_email_daily_limit(service, len(list(recipient_csv.get_rows())))
-        scheduled_for = datetime.fromisoformat(form.get("scheduled_for")) if form.get("scheduled_for") else None  # noqa: F821
+        scheduled_for = datetime.fromisoformat(data.get("scheduled_for")) if data.get("scheduled_for") else None  # noqa: F821
 
         if scheduled_for is None or not scheduled_for.date() > datetime.today().date():
             increment_email_daily_count_send_warnings_if_needed(
```

Comment:
I think the issue is on line 192.  If you pass `service` instead of `authenticated_service` I think everything will work as expected.

Reply:
🤦🏻‍♂️🤦🏻‍♂️🤦🏻‍♂️ Somehow I missed the fact that we already fetched the service here... thanks for catching this.